### DISCUSSION TITLE
0.8.5 is now LIVE!

### DISCUSSION THREAD

**Discussion by dayman40** [microsoft/WindowsAppSDK#1580] [Announcements]
## Windows App SDK 0.8.5 is now available

Windows App SDK 0.8.5 is a stable servicing release that continues to fix critical bugs for 0.8.0. 

- If you're building an app with the Windows App SDK for the first time, follow the [installation instructions here](https://docs.microsoft.com/en-us/windows/apps/winui/winui3/create-your-first-winui3-app?tabs=desktop-csharp).
- If you are already using the Windows App SDK 0.8 and would like to update to the latest release, you can do so by following the [update instructions here](https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/update-existing-projects-to-the-latest-release).

### Bug fixed in Windows App SDK 0.8.5:

- Fixed issue that was causing WinUI apps to crash.
- Fixed issue causing the title bar buttons (min, max, close) to not have rounded corners on Windows 11.
- Fixed issue causing the resizing layout options to not appear when hovering over maximize/restore button on Windows 11.
- Fixed issue causing a crashing exception where creating a [PointCollection](/windows/winui/api/microsoft.ui.xaml.media.pointcollection) object. For more information, see [issue 971](https://github.com/microsoft/CsWinRT/issues/971) on Github.

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v0.8.5'>0.8.5 is now LIVE! </a>.</em>

**Comment by Marv51** [comment]
Thank you for all your hard work everyone!

**Comment by JanRajnoha** [comment]
I have problem with upgrading to v0.8.5, because I'm getting error Duplicate page items, similar to v1.0.0-exp. And yes, I'm using community toolkit. Is this state okay or should be upgrade without problems?

**Reply by Marv51** [reply]
For me community toolkit 7.0.3 works great with 0.8.5. 
I had the same issue that you describe with the latest updates to "Microsoft.Xaml.Behaviors.WinUI.Managed" (2.0.4 is the latest that works) as they reference WinUI 1.0 preview.

**Reply by JanRajnoha** [reply]
oh, okay, I'll try and thanks for info

**Reply by JanRajnoha** [reply]
yeah, you were right. It's time to report it. Thank you again

**Comment by giraudremy** [comment]
Hi,

With version 0.8.5, I can't upload package to Microsoft Store, I have this error :
Votre package Microsoft.ProjectReunion.0.8 spécifie la version 8005.278.2204.0, alors que 8000.167.1906.0 est la version minimale disponible.

I have no problem with 0.8.4, why ?
